### PR TITLE
feat(undeploy) - Add an option that, when true, will also remove buil…

### LINKED ISSUE #190: unbuild command?
Currently we have undeploy,  this removes the application specific stuff that gets created during the apply-resource goal.

We don't have anything currently exposed that will remove the buildConfig, builds and Imagestreams that we create during the build phase.

there is the `build.recreate` option that a user can pass in to remove builds/imagestreams, but a build is run after that.

This would be more a method for our integration test framework, [rhoaster](https://github.com/bucharest-gold/rhoaster).  The code is there to delete these things, but it is inside of the build and imagstream files.  We would need to break that out into its own thing perhaps.

I don't really like the name "unbuild" so i'm open to suggestions.

### PR DESCRIPTION
**Pull request by lholmquist**
…ds, buildConfigs and Imagestreams

fixes #190

`nodeshift undeploy removeAll`

### CODE REVIEW

**Comment by lholmquist** [comment]
@lance @helio-frota so if we were to add this to rhoaster, undeploy(with the removeAll=true flag) should remove everything

**Comment by helio-frota** [comment]
@lholmquist sorry, I'm not 100% following the source code of rhoaster and nodeshift.
In some way this can help to have something more easy for rhoaster users to `undeploy-or-not` without need to comment the line  ` .then(_ => test.onFinish(testEnvironment.undeploy)) `  ?
If yes -> then LGTM 
if no and if this ^ makes no sense, nevermind my comment :smiley:

**Comment by lholmquist** [comment]
@helio-frota currently when undeploy happens, it just removes the deployment, route, service stuff. this would allow someone to also remove the builds and imagestreams that were created,  totally cleaning up everything.

File: bin/nodeshift [hunk 1]
```diff
@@ -84,6 +84,7 @@ yargs
   .argv;
 
 function commandHandler (argv) {
+  console.log(argv);
```
Comment (lance) [inline bin/nodeshift]: Do we need to log `argv` here?

Comment (lholmquist) [reply bin/nodeshift]: damn it :)

**Pull request merged**